api/v1/helper.go

```diff
@@ -292,6 +299,28 @@ func NeedToUpdateSriov(ifaceSpec *Interface, ifaceStatus *InterfaceExt) bool {
 							return true
 						}
 
+						if strings.EqualFold(ifaceStatus.LinkType, consts.LinkTypeETH) {
+							if vfStatus.Mac == consts.DefaultMAC {
```

Comment:
can you add a comment that we only do these checks if mac/guid contains a value ? (so that is clear it is intentional)

these fields in vf status may be empty because device is allocated to some workload.

Reply:
Added.

Reply:
also been thinking about it, i dont think MAC is never zero for a netdevice since its always set by the kernel.
@SchSeba do you agree ?

Reply:
Dropping this check. Sebastian let us know if you think otherwise.